File: Codeforces/707A.cpp
#include<iostream>
#include<string>
using namespace std;
int main()
{
	int n,m;
	string c;
	bool d=false;
	cin>>n>>m;
	for(int i=0;i<n;i++)
	{
		for(int j=0;j<m;j++)
		{
			cin>>c;
			if(c!="W" && c!="B" && c!="G")
			{
				cout<<"#Color"<<endl;
				d=true;
				break;
			}
		}
		if(d==true)
		break;
	}
	if(d==false)
	cout<<"#Black&White";
}

File: Codeforces/707A.py
n,m=list(map(int,input().split()))
for i in range(n):
    s=input().split()
    for j in s:
        if(j!="W" or j!="B" or j!="G"):
            print ("#Color")
            d=True
            break
    if(d==True):
        break
if(d==False):
    print ("#Black&White")
